src/main/kotlin/com/cjbooms/fabrikt/util/KaizenParserExtensions.kt

```diff
@@ -15,14 +16,14 @@ import java.net.URI
 
 object KaizenParserExtensions {
 
-    private val invalidNames =
-        listOf(
+    private val unsafePathSegments =
+        setOf(
             "anyOf",
             "oneOf",
             "allOf",
             "items",
             "schema",
-            "application~1json",
```

Comment:
It's a placeholder. The `pathFromRoot` function we use returns the yaml path separated by forward slashes. To avoid ambiguity, forward slashes in identifiers are replaced by `~1`. 
We now undo this replacement after splitting the path string by forward slashes, so we can use the original here.